resdk/tests/unit/test_resolwequerry.py

```diff
@@ -4,19 +4,25 @@
 # pylint: disable=missing-docstring, protected-access
 
 import unittest
+
 import six
+import slumber
 
 from mock import patch, MagicMock
 
 from resdk.resolwe import ResolweQuerry
 
-if six.PY2:
-    # pylint: disable=deprecated-method
-    unittest.TestCase.assertRegex = unittest.TestCase.assertRegexpMatches
-
 
 class TestResolweQuerry(unittest.TestCase):
 
+    @patch('resdk.resolwe.getattr')
+    @patch('resdk.resolwe.Resolwe')
+    @patch('resdk.resources.sample.Sample')
```

Comment:
In general, i understand that unit tests should cover only the function under test, not the whole package interaction... However, I agree that some interconnection can probably be good for bug recognition.

In this particular case, I just wanted a mock with "endpoint" attribute. I could created it manually, but using a real resource just seemed more explicit.